core/arch/arm/plat-rcar/conf.mk

```diff
@@ -21,6 +21,9 @@ $(call force,CFG_TEE_CORE_NB_CORE,8)
 CFG_TZDRAM_START ?= 0x44100000
 CFG_TZDRAM_SIZE	 ?= 0x03D00000
 CFG_TEE_RAM_VA_SIZE ?= 0x100000
+CFG_HWRNG_QUALITY ?= 1024
+CFG_HWRNG_PTA ?= y
+CFG_WITH_SOFTWARE_PRNG = n
```

Comment:
this statement has no strong effect. use either `?=` or `$(call force,...)`.
Here, I would recommend `CFG_WITH_SOFTWARE_PRNG ?= n`.

Reply:
dropped this statement altogether